lib/money.ex

```diff
@@ -517,12 +523,52 @@ defmodule Money do
     {:error, {Money.Invalid, "An amount must be specified but was not found in #{inspect(string)}"}}
   end
 
-  defp do_parse(%{"cb" => "", "ca" => currency, "amount" => amount}, _, options) do
-    maybe_create_money(currency, amount, options)
+  defp do_parse(
+         %{"cb" => "", "ca" => currency, "amount" => amount, "nb" => "", "na" => ""},
+         _,
+         options
+       ) do
+    maybe_create_money(currency, amount, false, options)
```

Comment:
Instead of all those additional pattern matches can't this just be:
```
defp do_parse(%{"cb" => "", "ca" => currency, "amount" => amount} = data, _, options) do
  maybe_create_money(currency, amount, data["nb"] == "-", options)
end
```
Maybe even with `data["nb"] == "-"` extracted in another named private function.

Reply:
Good point. To be honest I just copy-cat'ed the existing way of writing it out, to avoid making too many changes.